audit/projects/k8s-infra-e2e-boskos-002/iam.json

```diff
@@ -29,8 +29,7 @@
     },
     {
       "members": [
-        "group:k8s-infra-prow-oncall@kubernetes.io",
-        "user:spiffxp@google.com"
+        "group:k8s-infra-prow-oncall@kubernetes.io"
```

Comment:
This change is expected for all k8s-infra-e2e-* projects due to https://github.com/kubernetes/k8s.io/pull/1859#issuecomment-814364589

This addresses #299 for all k8s-infra-e2e-* projects